src/init.c

```diff
@@ -513,6 +513,13 @@ static void jl_uv_exitcleanup_walk(uv_handle_t *handle, void *arg)
 void jl_write_coverage_data(void);
 void jl_write_malloc_log(void);
 
+static struct uv_shutdown_queue_item *next_shutdown_queue_item(struct uv_shutdown_queue_item *item)
+{
+    struct uv_shutdown_queue_item *rv = item->next;
+    free(item);
+    return rv;
+}
```

Comment:
+1 to making valgrind happy if it doesn't cost us anything. There's also the consideration that someone could want to use `libjulia` and then unload it and get all their resources back, so this is a good practice.